adb_graphics/figures/maps.py

```diff
@@ -249,8 +249,10 @@ def _colorbar(self, cc, ax):
 
         if self.field.short_name == 'flru':
             ticks = [label.rjust(30) for label in ['VFR', 'MVFR', 'IFR', 'LIFR']]
+        if self.field.short_name == 'ulwrf' and self.field.level == 'sfc':
```

Comment:
not sure if would be useful here, but there is a way to set the number of tick labels in default_specs, I think it's:
- ticks > 0, set every "ticks" units, e.g., ticks=10, for temp, label every 10 deg.
- ticks=0, no tick labels
- ticks<0, label every "ticks" intervals., e.g. ticks=-2, label every other color level.

Reply:
I think that would be the best. We don't want multiple ways of doing the same things.